components/identities/actions/transferFormModal.tsx

```diff
@@ -32,34 +35,31 @@ const TransferFormModal: FunctionComponent<TransferFormModalProps> = ({
   const router = useRouter();
   const { address } = useAccount();
   const { tokenId } = router.query;
-  const numId = parseInt(tokenId as string);
   const { addTransaction } = useNotificationManager();
   const [addressInput, setAddressInput] = useState<string>("");
   const { starknetIdNavigator } = useContext(StarknetIdJsContext);
   const [isTxSent, setIsTxSent] = useState(false);
 
-  //set_domain_to_address execute
-  const transfer_identity_and_set_domain_multicall = [
-    {
-      contractAddress: process.env.NEXT_PUBLIC_NAMING_CONTRACT as string,
-      entrypoint: "set_domain_to_address",
-      calldata: [...callDataEncodedDomain, hexToDecimal(targetAddress ?? "")],
-    },
-    {
-      contractAddress: process.env.NEXT_PUBLIC_STARKNETID_CONTRACT as string,
-      entrypoint: "transferFrom",
-      calldata: [
-        hexToDecimal(address ?? ""),
-        hexToDecimal(targetAddress ?? ""),
-        numId,
-        0,
-      ],
-    },
-  ];
+  // todo: clear target address if necessary
+  // {
+  //   contractAddress: process.env.NEXT_PUBLIC_NAMING_CONTRACT as string,
+  //   entrypoint: "set_domain_to_address",
+  //   calldata: [...callDataEncodedDomain, hexToDecimal(targetAddress ?? "")],
+  // },
 
+  const transfer_id_call = {
```

Comment:
utils

Reply:
Done, again I moved it in `identityChangeCalls`.
See: https://github.com/starknet-id/app.starknet.id/pull/580/commits/bf4be29c54e76c88d43e045c23bf43a0c195e220